lib/http-outgoing.js

```diff
@@ -308,6 +330,10 @@ export default class PodletClientHttpOutgoing extends PassThrough {
     pushFallback() {
         // @ts-expect-error Internal property
         this.push(this.#manifest._fallback);
+        // @ts-expect-error Internal property
+        this.js = this.#manifest._js;
+        // @ts-expect-error Internal property
+        this.css = this.#manifest._css;
```

Comment:
We've fallen back so we grab the cached fallback js and css and set it on the outgoing.js and outgoing.css properties